packages/react/src/components/FeatureFlags/index.js

```diff
@@ -29,17 +29,37 @@ const FeatureFlagContext = createContext(GlobalFeatureFlags);
  * along with the current `FeatureFlagContext` to provide consumers to check if
  * a feature flag is enabled or disabled in a given React tree
  */
-function FeatureFlags({ children, flags = {} }) {
+function FeatureFlags({
+  children,
+  flags = {},
+  enableUseControlledStateWithValue = false,
```

Comment:
I don't think this flag is actually used anywhere and can be removed. https://github.com/carbon-design-system/carbon/pull/17239#discussion_r1729308202

Reply:
This still applies and can be removed